### DISCUSSION TITLE
how to display hover popup and   border

### DISCUSSION THREAD

**Discussion by ofzo** [helix-editor/helix#11163] [General]
<img width="740" alt="image" src="https://github.com/user-attachments/assets/8753f0b7-88db-4494-a46a-8cd1ae108ea1">

![image](https://github.com/user-attachments/assets/a9b08884-5310-4424-bd89-b27b97e99070)

**Comment by koibtw** [comment]
If by `hover popup and border` you mean `ui.popup` border (documented [here](https://docs.helix-editor.com/themes.html#interface)) that looks like this: (the red border)
![image](https://github.com/user-attachments/assets/d3ae7444-ad08-4f13-a9ed-627caff1c1e0)
, then I think there's no 'border' like that for `ui.menu`. You can always [create your own](https://docs.helix-editor.com/themes.html#creating-a-theme) version of the theme you're using and play with it or [open an issue](https://github.com/helix-editor/helix/issues/new?created_from_discussion_number=11163).
Command palette for example uses those text borders:
![image](https://github.com/user-attachments/assets/c0f53a54-5215-4a39-a6b4-0a91a15ae011)
